docs/production/deployment.md

```diff
@@ -45,6 +45,31 @@ extremely reliable for years, whereas the Docker image is new and has
 rough edges, so we recommend the normal installer unless you have a
 specific reason to prefer Docker.
 
+## Advanced Installer Options
+
+The Zulip installer supports the following advanced installer options
+as well as those mentioned in the [install](../production/install.html#installer-options)
+documentation:
+
+* `--postgresql-version`: Sets the version of PostgreSQL that will be installed.
+  We currently support PostgreSQL 10, 11, 12 and 13.
+
+* `--postgresql-missing-dictionaries`: Set `postgresql.missing_dictionaries`
+  ([docs][doc-settings]) in the Zulip settings, which alters the initial database.
+  This should be used with cloud managed databases like RDS. This option conflicts
+  with `--no-overwrite-settings`.
+
+* `--no-init-db`: This option instructs ther installer to not do any database
+  initialization. This should be used when you already have a Zulip database.
+
+* `--no-overwrite-settings`: This option preserves existing `/etc/zulip`
+  configuration files.
+
+* `--no-dist-upgrade`: This option makes the installer skip the initial
+  `apt-get dist-upgrade`. This can be useful if you need to speed the installer up,
+  however it should be noted that in doing this you may miss critical security patches
+  so it is not recommended unless you know how to run these manually.
```

Comment:
@hackerkid: We run this in the DO image -- is it accurate that we _will_ run `apt-get dist-upgrade` later in the process -- or do we rely on doing it during the daily cron jobs?

If we don't have any other use case for folks to run this, I'm not sure that we want to document this specialized of a footgun.